reference/auth/models/newappchallenge.v1.yaml

```diff
@@ -0,0 +1,10 @@
+title: New App Challenge
```

Comment:
nit: I wouldn't bother putting "new" in the filename or title... It isn't important, and can only confuse people (was there an older one?)